### DISCUSSION TITLE
Current status of Github Branches (May 2024)

### DISCUSSION THREAD

**Discussion by scottcwilson** [zencart/zencart#6502] [Announcements]
- The Github master branch is the destination for PRs targeted for the next release (2.1.0) 
- There *are* database changes in 2.1.0 vs 2.0.0/2.0.1, so you will want different databases if you are developing and testing against both releases.
- There are no current plans for a 2.0.2 release unless a serious issue is discovered.  If that happens, the release will be built off the  2.0 branch, and any PRs labelled "2.0.2" will be backported to that release.

Once 2.1.0 is released, we will create a new 2.1 branch, which will be for bugfixes, and master will become the branch for 2.2.

**Comment by brittainmark** [comment]
Has this happened, or is master still 2.1?

**Comment by scottcwilson** [comment]
Master is still 2.1.0 and will become 2.2.0.  No plans for a 2.1.1.
No definitive release schedule for 2.2.0; possibly late spring.

**Comment by brittainmark** [comment]
OK thanks so I assume it is ok to use master for a new install.

**Comment by scottcwilson** [comment]
An install for a test site?  Yes.
An install for a production site?  No.  You should use the latest released version, 2.1.0.
